translate/cloud-client/hybrid_glossaries/hybrid_tutorial.py

```diff
@@ -0,0 +1,257 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START hybrid_imports]
+# Imports the Google Cloud client libraries
+from google.cloud import translate_v3beta1 as translate
+from google.cloud import vision
+from google.cloud import texttospeech
+
+import io
+import os
+# [END hybrid_imports]
+
+# [START hybrid_project_id]
+# extract GCP project id
+PROJECT_ID = os.environ['GCLOUD_PROJECT']
+# [END hybrid_project_id]
+
+
+# [START hybrid_vision]
+def pic_to_text(infile):
+    # Detects text in an image file
```

Comment:
Here are some example Python docstrings for documenting functions including short description + arguments + return value (the info you're commenting)
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html